### PR TITLE
robo: add page

### PR DESCRIPTION
**Pull request by prudloff-insite**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/robo.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# robo
+
+> PHP task runner.
+> More information: <https://robo.li/>.
+
+- Run a specific command:
```
Comment (sbrl) [inline pages/common/robo.md]: Just want to clarify: Where does it find the tasks that it runs?

Comment (prudloff-insite) [reply pages/common/robo.md]: Tasks are defined in a file called `RoboFile.php` in the current folder.
I am not sure there is a way to specify another path for this file.

Comment (sbrl) [reply pages/common/robo.md]: Ah, I see. Thanks for clarifying :-)

**Comment by prudloff-insite** [comment]
`grumphp list` will list all the commands.

**Comment by sbrl** [comment]
Ah, cool @prudloff-insite. Perhaps we could add another example for that then please? It does sound useful enough to be included - probably before the current first example actually.

**Comment by sbrl** [comment]
Thanks, @prudloff-insite :smiley_cat:

**Pull request merged**